docs/pages/cv/classification_reference.md

```diff
@@ -0,0 +1,297 @@
+
+#### CV example
+
+```obo
+relationship: has_measurement_scope MS:4000XXX ! run level
+```
+
+### 4. Acquisition strategy — `applies_to_acquisition_mode`
+
+**Definition:**
+Specifies which acquisition mode or instrument configuration the metric is relevant for.
+
+#### Subclasses
+
+**Acquisition mode**
+
+* **Acquisition mode independent:** metrics valid for any acquisition method.
+* **Data-dependent acquisition (DDA):** metrics specific to stochastic precursor selection workflows.
+  *Example:* number of MS2 spectra per precursor.
+* **Data-independent acquisition (DIA):** metrics for window-based fragmentation strategies.
+  *Example:* precursor window purity.
+* **Targeted acquisition:** metrics for SRM, PRM, or other targeted workflows.
+  *Example:* transition reproducibility.
+* **Ion-mobility-coupled metric:** metrics derived from acquisition methods that include gas-phase ion mobility separation.
+  *Example:* TIMS mobility resolution (Δ1/K₀) per run.
+* **Imaging acquisition:** metrics for spatially resolved mass spectrometry experiments such as MALDI, DESI, or SIMS.
+  *Example:* pixel-to-pixel intensity variation across a tissue section.
+* **Other specialized mode:** metrics for advanced or hybrid acquisition modes such as BoxCar, MSⁿ, or multiplexed scanning.
+  *Example:* BoxCar intensity uniformity across boxes.
+
+**Instrument platform specificity**
+
+* **Orbitrap-specific:** metrics only applicable to Orbitrap instruments.
+  *Example:* Orbitrap transient length stability.
+* **TOF-specific:** metrics relevant to time-of-flight instruments.
+  *Example:* TOF detector voltage stability.
+* **Ion-trap-specific:** metrics specific to trap-based systems.
+  *Example:* Ion trap fill time distribution.
+* **Other platform-specific:** for quadrupoles, FT-ICR, or hybrid systems.
+
+#### CV example
+
+```obo
+relationship: applies_to_acquisition_mode MS:4000XXX ! acquisition mode independent
+```
+
+### 5. Quality interpretation type — `has_quality_directionality`
+
+**Definition:**
+Describes how a metric's numeric value relates to overall quality.
+This enables automatic reasoning about whether "higher," "lower," or "targeted" values represent better data.
+
+#### Subclasses
+
+* **Higher is better:** increasing values indicate improved quality.
+  *Example:* identification rate, mass accuracy score.
+* **Lower is better:** decreasing values indicate improved quality.
+  *Example:* FDR, mass error.
+* **Context dependent:** interpretation varies depending on method or range.
+  *Example:* precursor charge-state fractions, peak density.
+* **Target range:** optimal quality corresponds to values within a defined interval.
+  *Example:* temperature, pressure, retention-time drift.
+* **Categorical:** quality expressed as discrete categories (e.g., pass/fail, OK/warning/error).
+* **Trend:** metrics intended for temporal monitoring rather than direct ranking (e.g., instrument drift over time).
```

Comment:
I find `trend` would fit into one of the other categories, e.g. `instrument drift over time` fits into `lower is better`.
Are there any counter examples where this is not the case?